unit_tests/testAndersonAcceleration/testAndersonAcceleration.f90

```diff
@@ -339,7 +340,7 @@ SUBROUTINE testStep()
   ENDDO
 
   !Esure rate of convergence is correct:
-  ASSERT_APPROXEQ(AccErr(10)/AccErr(9),0.90109849144468168_SRK,'Wrong Anderson Conv. Rate')
+  ASSERT_APPROXEQ(AccErr(10)/AccErr(9),0.90463663552029550_SRK,'Wrong Anderson Conv. Rate')
```

Comment:
Why did this change?  I thought RAND and RANDOM_NUMBER were the same in 5.4?

Reply:
I thought that was the case, but apparently not. It could still be the same RNG under the hood in 5.4 but the seed could have been different. I changed it to force the random seed to be 0 at the beginning just to be thorough.

Reply:
Alright, yeah, definitely in favor of fixing the seed to 0.  I'm good with letting this ride.  If we happen to have issues again then I think we switch to Futility's StochasticSampler package, but I'd rather not do that if we can help it.

Reply:
whats wrong with StochasticSampler??

Reply:
Maybe nothing.  But the versions Jesse uses here should be fixed by the standard, and technically StochasticSampler could be changed by future work.  I'd just as soon have as few moving parts as possible.

Reply:
The interface is fixed, but do we know that Intel doesn't use a different seed, or a completely different sampling method?  Sampler will allow you to choose the RNG type and the seed so that you know its portable.  The standard doesn't say what the RNG methodology needs to be so I don't think you can count on the output being standardized

Reply:
also, it says esure instead of ensure in the comment